src/components/design_system/button/Button.module.css

```diff
@@ -57,6 +57,27 @@
   background-color: var(--primary-container);
 }
 
+.tertiary {
+  display: block;
+  color: var(--primary-50);
+  padding: 10px 12px;
+  border-radius: 8px;
+  cursor: pointer;
+  text-decoration: underline;
+  text-align: center;
+  font-size: 16px;
```

Comment:
I would advise having font size as either `1 rem` or the global size of `var(--h6)` so that it's scalable with smaller screens.

Reply:
Great call, @hieungo89 ! I reused the code for the secondary button and don't know why that became pixels. The secondary button font size is var(--base-size). Would that work?